### PR TITLE
Task/187 upgrade to api33

### PR DESCRIPTION
**Pull request by MariooW**
Update local-api to versions 32 and 33
Changed version of JAVA to 11 on android tests

### CODE REVIEW

**Review by TomasPetrik — APPROVED** [review]

File: .github/workflows/_test_application.yml [hunk 1]
```diff
@@ -145,7 +150,10 @@ jobs:
...
         with:
           api-level: ${{ matrix.api-level }}
+          disk-size: 4096M
+          heap-size: 1024M
+          sdcard-path-or-size: 1000M
           target: ${{ matrix.target }}
           arch: x86_64
           profile: Nexus 6
-          script: flutter test integration_test --reporter expanded --flavor dev --dart-define=TEST_RUN=true
+          script: flutter test integration_test --reporter expanded --flavor dev --dart-define=TEST_RUN=true
```
Comment (jvrastil) [inline .github/workflows/_test_application.yml:159]: can you remove unnecessary line below?

File: .github/workflows/run_tests_on_push_test_branch.yml [hunk 2]
```diff
@@ -4,11 +4,12 @@ on:
   push:
     branches:
       - 'test/**'
+      - 'task/**'
```
Comment (jvrastil) [inline .github/workflows/run_tests_on_push_test_branch.yml:7]: remove duplicated rules please

Comment (jvrastil) [reply .github/workflows/run_tests_on_push_test_branch.yml:7]: oh my mistake

File: .github/workflows/_test_application.yml (hunk 1 above)
Comment (jvrastil) [reply .github/workflows/_test_application.yml:159]: ok, it looks good

**Comment by MariooW** [comment]
After re-base showed that the files were changed even when they weren't

**Pull request closed without merging**